.github/workflows/actions.yml

```diff
@@ -20,7 +20,11 @@ jobs:
       uses: actions/checkout@v1
 
     - name: Get Dependencies using Go Modules
-      run: go mod download
+      run: make dependency
 
-    - name: Build Coderunner
-      run: go build -o coderunner
+    - name: Build Server
+      run: make
+      
+    - name: Test server
+      run: ./code-runner-server -port '8083' & go test ./... -port '8083'
```

Comment:
Use the default port as there is no valid reason for using a port other than the default one (8082).

Reply:
Yes. That's unnecessary. Done